src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,47 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div
+    className="card"
+    data-cy="Movie"
+  >
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt={`${movie.tittle} logo`}
```

Comment:
You have a typo in alt tag for image. It should be `movie.title` instead of `movie.tittle`